app/views/trainees/lead_schools/_no_results.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1 class="govuk-heading-l govuk-!-margin-bottom-1">
```

Comment:
My latest PR also has a `no_results` partial including a search field. We're definitely on the same wavelength. :-)